__tests__/app.test.js

```diff
@@ -15,4 +15,12 @@ describe('GET /api/categories', () => {
             expect(Array.isArray(res.body)).toBe(true);
         });
     });
+    it('returns an array of categories with slug and description properties', () => {
+        return request(app).get('/api/categories')
+        .expect(200)
+        .then((res) => {
+            expect(res.body[0].hasOwnProperty("slug")).toBe(true);
```

Comment:
We'll just have to tweak this test once you change the response to be returned in an object.